app/services/cart.js

```diff
@@ -180,6 +181,7 @@ export default ApiService.extend({
    * If the user is not logged in, will only save it locally
    */
   addPackage(pkg) {
+    let _this = this;
```

Comment:
No need of creating `_this`. It will be handled since `arrow` function is used in `catch` block.